oic.r.session.raml

```diff
@@ -0,0 +1,45 @@
+#%RAML 0.8
+title: Account Session
+version: v1.0-20180320
+documentation:
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+schemas:
+  - Account-Session-Request: !include schemas/oic.r.session-request.json
+  - Account-Session-Response: !include schemas/oic.r.session-response.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.baseline"]
+
+/oic/sec/session:
+  description: |
+    Resource that enables a device to manage its session using login or logout
+
+  post:
+    body:
+      application/json:
+        schema: Account-Session-Request
+        example: |
+          {
+            "uid" : "123e4567-e89b-12d3-a456-d6e313b71d9f",
+            "di" : "9cfbeb8e-5a1e-4d1c-9d01-00c04fd430c8",
+            "accesstoken" : "0f3d9f7fe5491d54077d",
+            "login" : true
+          }
+    responses:
+      204:
+        body:
+          application/json:
+            schema: Account-Session-Response
+            example: |
+              {
+                "expiresin" : 3600
+              }
```

Comment:
Add information about a RETRIEVE/Get to align with this:

The RETRIEVE operation on OCF Cloud’s “/oic/sec/session” Resource is not allowed and the OCF Cloud is expected to reject all attempts to perform such operation.

Reply:
Currently "get" is not defined in this schema, which should mean this operation is not supported. @marktrayer , can you also please provide inputs on this, if we need to add anything.